src/internal.c

```diff
@@ -2388,6 +2388,21 @@ int InitSSL_Ctx(WOLFSSL_CTX* ctx, WOLFSSL_METHOD* method, void* heap)
     ret = wolfEventQueue_Init(&ctx->event_queue);
 #endif /* HAVE_WOLF_EVENT */
 
+#if defined(HAVE_PK_CALLBACKS) && defined(WOLFSSL_MAXQ10XX_TLS)
+    /* Only setup the callbacks if we're a client. Let the maxq10xx
+     * infrastructure know whether we are doing TLS 1.3. */
+    if (method->side == WOLFSSL_CLIENT_END) {
+        ctx->devId = MAXQ_DEVICE_ID;
+        if (method->version.major == SSLv3_MAJOR &&
+            method->version.minor == WOLFSSL_TLSV1_3) {
+            maxq10xx_SetupPkCallbacks(ctx, 1);
```

Comment:
Would it be easier -> more future proof to just provide the method->version here?

Reply:
YES!! Stay tuned for that.

Reply:
Fixed by https://github.com/anhu/wolfssl/pull/9/commits/89b837a6d9312623af8ebb96218c7f6a6805c55d